src/Symfony/Component/PasswordHasher/Hasher/UserPasswordHasher.php

```diff
@@ -32,30 +32,20 @@ public function __construct(PasswordHasherFactoryInterface $hasherFactory)
 
     public function hashPassword(PasswordAuthenticatedUserInterface $user, string $plainPassword): string
     {
-        $salt = null;
-        if ($user instanceof LegacyPasswordAuthenticatedUserInterface) {
```

Comment:
this change will increase merge conflicts for no benefits IMO

Reply:
agreed, reverted